src/tests/baseservices/exceptions/unhandledHandler/ForeignThreadRevPInvokeUnhandled/ForeignThreadRevPInvokeUnhandledNative.cpp

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "stdio.h"
+#include <stdlib.h>
+
+#ifdef _WIN32
+#pragma warning(push)
+#pragma warning(disable:4265 4577)
+#include <thread>
+#pragma warning(pop)
+#else // _WIN32
+#include <pthread.h>
+#endif // _WIN32
+
+// Work around typedef redefinition: platformdefines.h defines error_t
```

Comment:
I wonder if the `platformdefines.h` should be fixed instead to not to define the `error_t` and include the `errno.h` instead.